cmd/ketch/root.go

```diff
@@ -57,6 +57,8 @@ func newRootCmd(cfg config, out io.Writer, packSvc *pack.Client, ketchConfig con
 			return cmd.Usage()
 		},
 	}
+
+	cmd.PersistentFlags().StringP("output", "o", "", "used to specify output, e.g. --output format=json")
```

Comment:
Does this technically count as a global command? From the documentation, this will be available on all commands, but should this be expected to work on something like `ketch framework add`?

Reply:
This is a good call. I went back & forth on whether to add output formatting to every command (`{"status":"Successfully added framework!"}`?) I moved the flag to the affected commands.